src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,155 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public MyLinkedList() {
+    }
+
+    public MyLinkedList(List<T> list) {
+        this();
+        addAll(list);
+    }
```

Comment:
let's better remove these constructors at the moment

Reply:
done